packages/patternfly-4/react-charts/src/components/ChartThreshold/ChartThreshold.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+import { VictoryLine } from 'victory';
+import { ChartLine, ChartLineProps } from '../ChartLine';
+import { getThresholdTheme } from '../ChartUtils/chart-theme';
+
+export const ChartThreshold: React.FunctionComponent<ChartLineProps> = ({
+  themeColor,
+  themeVariant,
+  style, // do not let style defaults override threshold theme
+  ...rest
+}: ChartLineProps) => {
+  const theme = getThresholdTheme(themeColor, themeVariant);
+  return (
```

Comment:
when extending ChartLineProps this way, it seems `style` prop will override `theme`. This method works for allowing theme colors, but would prevent the consumer from override `style` and `theme` directly. Any suggestions @dlabrecq ? I think this is close to what we discussed...

Reply:
I believe this is expected behavior. For example, when overriding the `padding` prop, users must provide all variables; top, bottom, left, right.